include/spirv/unified1/spirv.core.grammar.json

```diff
@@ -11061,6 +11122,12 @@
           "enumerant" : "SubpassData",
           "value" : 6,
           "capabilities" : [ "InputAttachment" ]
+        },
+        {
+          "enumerant" : "TileImageDataEXT",
+          "value" : 4173,
+          "capabilities" : [ "TileImageColorReadAccessEXT" ],
+          "version" : "None"
```

Comment:
I was not sure if I should add a version field here or not (since these enumerants haven't been extended before). I saw the StorageClass list above did include it so did the same, though the ImageFormat further down does not. Let me know if there's a preference for having it here vs not.